### PR TITLE
Add org.gnome.Radio

### PR DESCRIPTION
**Pull request by oleaamot**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by oleaamot** [comment]
bot, build org.gnome.Radio

**Comment by hfiguiere** [comment]
What's the relation with #3559 ?

Also the comments on #3559 applies here.

File: org.gnome.Radio.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            "sources": [
+		{
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/cogl/1.22/cogl-1.22.8.tar.xz",
+                    "sha256": "a805b2b019184710ff53d0496f9f0ce6dcca420c141a0f4f6fcc02131581d759"
+		}
+            ]
+	},  
+	{
+            "name": "clutter",
```
Comment (hfiguiere) [inline org.gnome.Radio.json:44]: you should use clutter from the shared modules.

File: org.gnome.Radio.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
+{
+  "app-id" : "org.gnome.Radio",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "43",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "gnome-radio",
+  "finish-args" : [
+     "--socket=x11",
+     "--share=ipc"
```
Comment (hfiguiere) [inline org.gnome.Radio.json:9]: you enable wayland everywhere but don't allow it here. Also DRI is probably needed too. And I don't see something for the sound.

File: org.gnome.Radio.json [hunk 3]
```diff
@@ -0,0 +1,105 @@
+{
+  "app-id" : "org.gnome.Radio",
```
Comment (hfiguiere) [inline org.gnome.Radio.json:2]: See https://github.com/flathub/flathub/pull/3559#issuecomment-1358686987

**Pull request closed without merging**